packages/components-react/projects/react-wrapper/tests/unit/specs/usePrefix.spec.ts

```diff
@@ -0,0 +1,52 @@
+import { skipCheckForPorscheDesignSystemProviderDuringTests, usePrefix } from '../../../src/hooks';
+import * as React from 'react';
+import { PorscheDesignSystemContext } from '../../../src/provider';
+
+// Test is in a separate file as this mocks useContext() for the whole test-suit
+jest.mock('react', () => {
+  const ActualReact = jest.requireActual('react');
+  return {
+    ...ActualReact,
+    useContext: jest.fn().mockReturnValue({ prefix: 'my-prefix' }), // what you want to return when useContext get fired goes here
+  };
+});
+
+describe('usePrefix()', () => {
+  const originalEnv = process.env;
+
+  afterEach(() => {
+    process.env = originalEnv;
+  });
+
+  it('should return tagName if process.env.NODE_ENV is set to "test" and skipCheck is true', () => {
```

Comment:
`tagName` might be misleading in this context.

Reply:
Extended description with "passed tagName" to clarify that it is about the tagName parameter of the function